core/domain/collection_services.py

```diff
@@ -555,28 +555,14 @@ def get_collection_ids_matching_query(query_string, cursor=None):
         collection_ids, search_cursor = search_services.search_collections(
             query_string, remaining_to_fetch, cursor=search_cursor)
 
-        invalid_collection_ids = []
-        for ind, model in enumerate(
+        for ind, _ in enumerate(
                 collection_models.CollectionSummaryModel.get_multi(
                     collection_ids)):
-            if model is not None:
```

Comment:
The model cannot be None as we are fetching the collection ids through query and there cannot be a collection id for which there is no collection

Reply:
Done